server/graphql/schema/Mutation.js

```diff
@@ -3,6 +3,7 @@ const { mutationType } = require('nexus');
 const Mutation = mutationType({
   definition(t) {
     t.crud.createOneUser();
+    t.crud.createOnePost();
```

Comment:
This allow us to send ANY piece of information regarding the post. Might not be ideal.